pkgs/applications/science/machine-learning/starspace/default.nix

```diff
@@ -0,0 +1,29 @@
+{lib, stdenv, fetchFromGitHub, boost165}:
```

Comment:
```suggestion
{ lib, stdenv, fetchFromGitHub, boost165 }:
```

Reply:
Applied in f8b69fa6d2fcf54548795e3e04845f304efb3a30